client/android/app/build.gradle

```diff
@@ -51,6 +53,19 @@ android {
             signingConfig signingConfigs.debug
         }
     }
+
+    flavorDimensions app
```

Comment:
`app` is too ambiguous, so change to server instead:
https://developer.android.com/studio/build/build-variants#flavor-dimensions
```
flavorDimensions "server"
```